pkg/transforms/argoapplication.go

```diff
@@ -192,13 +196,26 @@ func (a ArgoApplicationResource) BuildEdges(ns NodeStore) []Edge {
 						DestUID:    destNode.UID,
 						DestKind:   resource.Kind,
 					})
-				} else {
-					glog.Infof("For %s, subscribesTo edge not created as %s named %s not found",
-						resource.Kind+"/"+namespace+"/"+resource.Name, resource.Kind, namespace+"/"+resource.Name)
 				}
+			} else {
+				glog.Warningf("For %s, subscribesTo edge not created as %s named %s not found",
+					resource.Kind+"/"+namespace+"/"+resource.Name, resource.Kind, namespace+"/"+resource.Name)
+
+				missingResc = append(missingResc, resource)
 			}
 		}
 	}
 
+	// add missing application resource as a property
+	if len(missingResc) != 0 {
+		rescb, err := json.Marshal(missingResc)
+
+		if err != nil {
+			glog.Error("failed to marshal resource", err)
```

Comment:
Please add more details to this error message.
```suggestion
			glog.Error("AgroApplication transform failed to marshal missingResc", err)
```